app/src/main/java/com/example/keybindhelper/Dialogs/GroupListProvider.java

```diff
@@ -47,10 +47,10 @@ public GroupListProvider(Context c, String title, List<Group> groups){
             scrollLine.addView(b);
         }
         if(groups.size()==0){
-            TextView noitems=new TextView(c);
-            noitems.setText("None");
-            noitems.setTextAlignment(View.TEXT_ALIGNMENT_CENTER);
-            line.addView(noitems);
+            TextView noItems=new TextView(c);
+            noItems.setText("None");
+            noItems.setTextAlignment(View.TEXT_ALIGNMENT_CENTER);
+            line.addView(noItems);
```

Comment:
Added camelcase to noItems to refer Kotlin best naming practices.

Reply:
Good suggestion.